cypress/support/index.js

```diff
@@ -63,47 +68,23 @@ before(() => {
 
 Cypress.Screenshot.defaults({ screenshotOnRunFailure: DEBUG_FLAG });
 
-// Configure custom commands eyes-cypress
-import '@applitools/eyes-cypress/commands'
-
 const {
   Before,
   After
 } = require("cypress-cucumber-preprocessor/steps");
 
 
-Before({ tags: "@applitools" }, () => {
-  // debugger
-
-  cy.eyesOpen({
-    appName: 'Carbon Sage',
-    matchLevel: 'Strict', // Strict, Exact, Layout and Content - https://help.applitools.com/hc/en-us/articles/360007188591-Match-Levels
-    envName: 'Development',
-    testName: cy.state('ctx').test.title, // Assign the current test name to Applitools test name
-    batchName: cy.state('ctx').test.parent.title, // Assign the feature name to Applitools batch name
-    browser: [            
-              // {width: 800, height: 600, name: 'firefox'},
-              // {width: 1024, height: 768, name: 'chrome'},
-              {width: 1440, height: 1219, name: 'edge'}
-
-              // {
-              //     deviceName: 'iPhone X',
-              //     screenOrientation: 'landscape',
-              //     name: 'chrome' // optional, just to make it explicit this is browser emulation and not a real device. Only chrome is supported for device emulation.
-              // },
-
-              // {
-              //  deviceName: 'iPad',
-              //   screenOrientation: 'landscape',
-              //   name: 'chrome' // optional, just to make it explicit this is browser emulation and not a real device. Only chrome is supported for device emulation.
-              // }
-            ]
-  
-  }); 
-});
+if ( Cypress.env('CYPRESS_APPLITOOLS') ) {
+  Before({ tags: "@applitools" }, () => {
+    applitools_settings.testName = cy.state('ctx').test.title
+    applitools_settings.batchName = cy.state('ctx').test.parent.title
```

Comment:
why we can't set those settings in applitools  config ?

Reply:
`cy.state('ctx').test.title` is out of context in applitools config file

Reply:
Could only get this working with in the Before